src/test/java/teammates/test/cases/action/InstructorFeedbackAddActionTest.java

```diff
@@ -219,11 +223,13 @@ protected InstructorFeedbackAddAction getAction(String... params) {
         return (InstructorFeedbackAddAction) gaeSimulation.getActionObject(getActionUri(), params);
     }
 
-    protected String getPageResultDestination(
-            String parentUri, String courseId, String fsname, String userId, boolean isError) {
+    protected String getPageResultDestination(String parentUri, String courseId, String fsname,
+            boolean isToBeLoadedInEditMode, String userId, boolean isError) {
```

Comment:
The good thing about allowing the param to be null is that all these changes are unnecessary, as a null param is interpreted as false. All the changes to this file can be reverted :)